server/src/share-buttons.js

```diff
@@ -155,10 +155,10 @@ class ShareButtonPanel extends React.Component {
           value={ this.props.shot.viewUrl }
           onClick={ this.onClickInputField.bind(this) }
           onChange={ function() {} /* react gives a warning otherwise */ } />
-        <Localized id={ this.state.copy === "copy" ? "shotPageCopy" : "shotPageCopied" }>
+        <Localized id={ this.state.copy === "copy" ? "shotPageCopyText" : "shotPageCopied" }>
```

Comment:
I'm confused. What was "shotPageCopy" before this PR?

Reply:
@flodolo that was for exact same text "Copy" but got removed in a PR that landed for new copy button on shots. 
Bringing it back with new id as we still need it on my shots page or can we still use the same id in this case?